app/util/process_utils.py

```diff
@@ -27,6 +27,14 @@ def kill_gracefully(process, timeout=2):
     return process.returncode, stdout, stderr
 
 
+def _is_windows():
+    """
```

Comment:
I am leaving this method 'private' to this package for now since it's only being used in this package at this moment. We can totally move it to a better place if there are more usages.